xbmc/storage/cdioSupport.h

```diff
@@ -29,8 +29,6 @@
 
 #include "system.h" // for HAS_DVD_DRIVE
```

Comment:
The comment is now wrong, include is needed for `ssize_t` typedef.
I think including `PlatformDefs.h` would be enough.

Reply:
let me try :D